tgui/.eslintrc.yml

```diff
@@ -651,7 +651,7 @@ rules:
   ## Enforce ES5 or ES6 class for React Components
   react/prefer-es6-class: error
   ## Enforce that props are read-only
-  react/prefer-read-only-props: error
+  react/prefer-read-only-props: off
```

Comment:
we don't use this anywhere. it absolutely goes off now and will try to fix the type by adding `readonly` to every prop, which is the messiest way. 

this should simply be disabled. it's pretty unusual for someone to try to mutate props. if this case exists the entire type should be wrapped in `Readonly<x>`